src/optimize-streams.ts

```diff
@@ -102,19 +104,23 @@ export class GenericOptimizeTransform extends Transform {
  * Transform JavaScript.
  */
 export class JsTransform extends GenericOptimizeTransform {
-  constructor(options: JsOptimizeOptions) {
+  constructor(options: OptimizeOptions) {
+    const jsOptions = options.js || {};
```

Comment:
🤡🏌️: `const {jsOptions = {}} = options;`

Reply:
I think that would be `const {js: jsOptions = {}} = options;` which is slightly longer and probably more confusing than the original?